src/types/onyx/ReimbursementAccountDraft.ts

```diff
@@ -55,7 +64,19 @@ type ReimbursementAccountProps = {
     amount3?: string;
 };
 
-type ReimbursementAccountDraft = BankAccountStepProps & CompanyStepProps & RequestorStepProps & ACHContractStepProps & ReimbursementAccountProps;
+// BeneficialOwnerDraftData is saved under dynamic key which consists of prefix, beneficial owner ID and input key
+type BeneficialOwnerDraftData = Record<`beneficialOwner_${string}_${string}`, string>;
```

Comment:
Wdyt to extract `beneficialOwner_` to a string or a type?

Reply:
cc @MrMuzyk

Reply:
so you want separate type for the whole template string? 
`
type BeneficialOwnerDataKey = `beneficialOwner_${string}_${string}`
`

Im ok with it